package.json

```diff
@@ -68,6 +70,7 @@
     "@storybook/theming": "^7.1.1",
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^14.0.0",
+    "@types/aria-query": "^5.0.2",
```

Comment:
I don't think it's needed here - what for?

Reply:
Initially added @types/aria-query to resolve type issues, but the problem resolved itself.